channels/disp/client/CMakeLists.txt

```diff
@@ -19,7 +19,9 @@ define_channel_client("disp")
 
 set(${MODULE_PREFIX}_SRCS
 	disp_main.c
-	disp_main.h)
+	disp_main.h
+	../disp_common.c
```

Comment:
Could split the common parts to a `CMake` object library

Reply:
Hmm, can you please explain? I actually did it the same way it was done in rdpgfx

Reply:
as the common part is used by both, client and server, it could be handled as `static` lib (or better `object` lib) to be compiled once and used by the respective channel.
this is a 'nice to have' thing though, and as you said, we have not done it in all places ourselves yet.

Reply:
Hmm, okay, so I'll be happy if we merge and maybe I'll open another PR for this (sorry, I have a lot of other things to work on)